package.json

```diff
@@ -46,6 +46,7 @@
     "@web-stories-wp/stories-lightbox": "*",
     "@web-stories-wp/templates": "*",
     "@web-stories-wp/text-sets": "*",
+    "@web-stories-wp/tinymce-buttons": "*",
```

Comment:
Why plural `buttons`? I thought it's only 1 button?

Reply:
Oh, read your feedback as buttons. May we have other buttons in the future?

Reply:
I doubt it. Let's keep it singular.